DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/TutorialViewModelTests.cs

```diff
@@ -329,7 +388,8 @@ public void Tutorial_should_decide_to_show_progress()
             // Given
             var expectedTutorialInformation = TutorialContentHelper.CreateDefaultTutorialInformation(
                 canShowDiagnosticStatus: true,
-                attemptCount: 1
+                attemptCount: 1,
+                courseSettings: null

```

Comment:
This is the default, I thought it would be helpful to add it here in case someone didn't know what the default was, considering the course settings can have an impact on this value. I can remove them if you like? 🙂

Reply:
I think this is a natural default to have, and it's quite easy to check if the person looking at it isn't sure. So I wouldn't mind if you remove them. But I don't feel very strongly about it so feel free to keep if you prefer 🙂

Reply:
Yeah fair enough, I've removed  them in ac967b7 🙂